sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryListener.scala

```diff
@@ -96,7 +96,8 @@ object StreamingQueryListener {
   class QueryStartedEvent private[sql](
       val id: UUID,
       val runId: UUID,
-      val name: String) extends Event
+      val name: String,
+      val trigger: Trigger = ProcessingTime(0L)) extends Event
```

Comment:
Do we need this default value? Is it set to avoid compilation failures at other usages(scala tests)?

Reply:
Yes. That is true.